HotelReservation/src/main/java/HotelList.java

```diff
@@ -0,0 +1,43 @@
+import java.util.ArrayList;
+
+public class HotelList {
+    private ArrayList<Hotel> listOfHotels;
+
+    HotelList(){
+        listOfHotels = new ArrayList<Hotel>();
+    }
+    public void addHotel(Hotel h){
+        listOfHotels.add(h);
+    }
+
+    public void removeHotel(Hotel h){
+        listOfHotels.remove(h);
+    }
+
+    public int size(){
```

Comment:
change this api to get the count of hotels, size is not clarifying what size it is